diff --git a/node_modules/ngx-awesome-uploader/lib/file-picker.component.d.ts b/node_modules/ngx-awesome-uploader/lib/file-picker.component.d.ts
index 5b9dda5..c04833b 100644
--- a/node_modules/ngx-awesome-uploader/lib/file-picker.component.d.ts
+++ b/node_modules/ngx-awesome-uploader/lib/file-picker.component.d.ts
@@ -11,7 +11,7 @@ import { HttpErrorResponse } from '@angular/common/http';
 import * as i0 from "@angular/core";
 export declare class FilePickerComponent implements OnInit, OnDestroy {
     private fileService;
-    private changeRef;
+    public changeRef;
     /** Emitted when file upload via api successfully. Emitted for every file */
     readonly uploadSuccess: EventEmitter<FilePreviewModel>;
     /** Emitted when file upload via api failed. Emitted for every file */
